src/sentry/migrations/0053_remove_old_functions.py

```diff
@@ -0,0 +1,132 @@
+        )
+
+    yAxis = old_query.get("yAxis")
+    if yAxis:
+        new_query["yAxis"] = convert_function(yAxis)
+
+    fields = old_query.get("fields")
+    new_fields = []
+    for field in fields:
+        new_fields.append(convert_function(field))
+    new_query["fields"] = new_fields
+
+    search = old_query.get("query")
+    if search:
+        match = COUNT_REGEX.match(search)
+        if match:
+            search = search.replace(match.groups()[0], "count()")
+        for old_fn, new_fn in six.iteritems(FUNCTION_CHANGE):
+            if old_fn + "()" in search:
+                search = search.replace(old_fn + "()", new_fn)
+            elif old_fn in search:
+                search = search.replace(old_fn, new_fn)
+        new_query["query"] = search
+
+    DiscoverSavedQuery.objects.filter(id=saved_query.id).update(query=new_query)
+
+
+def migrate_functions_in_queries(apps, schema_editor):
+    """
+    Creates v2 versions of existing v1 queries
+    """
+    DiscoverSavedQuery = apps.get_model("sentry", "DiscoverSavedQuery")
+
+    """
+    Seq Scan on sentry_discoversavedquery (cost=0.00..225.15 rows=1077 width=200) (actual time=0.054..7.875 rows=1037 loops=1)
+    Filter: ((version = 2) AND ((query ~~ '%p95%'::text) OR (query ~~ '%p99%'::text) OR (query ~~ '%p75%'::text) OR (query ~~ '%apdex%'::text) OR (query ~~ '%impact%'::text) OR (query ~~ '%last_seen%'::text) OR (query ~~ '%latest_event%'::text) OR (query ~~ '%count(%'::text)))
+    Rows Removed by Filter: 2074
+    Planning time: 2.305 ms
+    Execution time: 8.694 ms
+    """
+    function_filter = Q(query__contains="count(")
+    for key in FUNCTION_CHANGE:
+        function_filter = function_filter | Q(query__contains=key)
+
+    queryset = DiscoverSavedQuery.objects.filter(function_filter, version=2)
+
+    for query in RangeQuerySetWrapper(queryset):
+        convert(DiscoverSavedQuery, query)
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn't be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = False
+
+    # This flag is used to decide whether to run this migration in a transaction or not.
+    # By default we prefer to run in a transaction, but for migrations where you want
+    # to `CREATE INDEX CONCURRENTLY` this needs to be set to False. Typically you'll
+    # want to create an index concurrently when adding one to an existing table.
+    atomic = True
```

Comment:
Make this `False` plz.

@wedamija we might wanna update the comment h ere to suggest that data migrations should pretty much never be run atomically.

Arguably, I think we should change this default to be False and explicitly opt in when we want it. Any atomic transactions are pretty risky if they take measurable amounts time.